src/BloomExe/Book/Book.cs

```diff
@@ -1704,8 +1704,11 @@ public void DuplicatePage(IPage page)
 
 			ClearPagesCache();
 			Save();
-			if (_pageListChangedEvent != null)
-				_pageListChangedEvent.Raise(null);
+			// The only caller calls _view.UpdatePageList(false); immediately after this method returns.
```

Comment:
So you could remove the Raise() from the called instead. In doing that, you'd leave DuplicatePage parallel with the the calls for InsertPageAfter() and DeletePage().